PryntTrimmerView/Classes/Trimmer/PryntTrimmerView.swift

```diff
@@ -86,55 +159,31 @@ public protocol TrimmerViewDelegate: AVAssetTimeSelectorDelegate {
             assetPreview.maxOnscreenDuration = maxOnscreenDuration
         }
     }
-
-    private var lastWidth: CGFloat?
-
-    // MARK: - View & constraints configurations
-
-    override public func layoutSubviews() {
-        super.layoutSubviews()
-
-        let newWidth = bounds.width
-        guard let lastWidth = lastWidth else {
-            self.lastWidth = newWidth
-            return
+    
+    public var fullWidth: CGFloat { assetPreview.contentWidth + 2 * handleWidth }
```

Comment:
1. Add const
2. Use ()

```suggestion
    public var fullWidth: CGFloat { assetPreview.contentWidth + (someConst * handleWidth) }
```